src/Angor/Avalonia/Angor.Contexts.Funding/Projects/Infrastructure/Interfaces/IProjectAppService.cs

```diff
@@ -8,6 +8,7 @@ public interface IProjectAppService
 {
     Task<Result<IEnumerable<ProjectDto>>> Latest();
     Task<Maybe<ProjectDto>> FindById(ProjectId projectId);
+    public Task<Result<ProjectDto>> Get(ProjectId projectId);
```

Comment:
No need for public, also why do we have findById and Get? can't we use just one for the same logic?